### PR TITLE
Made support-email configurable, fixes #100

### LINKED ISSUE #100: Supportmail is hardcoded
The mailing adress for support (kasse@fablab.fau.de) is hardcoded. Search all occurences and replace by config-option.

### PR DESCRIPTION
**Pull request by sedrubal**
- needs testing, because it would be bad if no error message will be displayed, because of such a trivial mistake, but I don't know how to trigger suitable errors, besides the `DebtLimitExceeded` exception - there it works...
- is vulnerable to #63

### CODE REVIEW

**Comment by sedrubal** [comment]
@patkan but I don't know how to do it...

**Comment by patkan** [comment]
@sedrubal I was just too fast with assigning, the commit should fix your problem ;-)

**Comment by patkan** [comment]
Seems to be ok.
@sedrubal you could rebase this PR to one commit and then we could merge it.

File: FabLabKasse/scriptHelper.py [hunk 1]
```diff
@@ -64,11 +64,12 @@ def myNewExceptionHook(exctype, value, tb):
         import datetime
         # logging.exception()
         try:
+            cfg = getConfig()
+            email = cfg.get('general', 'support_mail')
```
Comment (mgmax) [inline FabLabKasse/scriptHelper.py]: please catch ConfigParser.Error here so that the crash handler can't crash

**Comment by patkan** [comment]
Sorry, had to rebase due to sloppiness.
This should work now.

File: FabLabKasse/scriptHelper.py [hunk 2]
```diff
@@ -64,11 +65,16 @@ def myNewExceptionHook(exctype, value, tb):
         import datetime
         # logging.exception()
         try:
+            cfg = getConfig()
+            try:
+                email = cfg.get('general', 'support_mail')
+            except ConfigParserError:
+                logging.warning("could not read mail address from config in graphical except-hook.")
+                email = "dex Verantwortlichx"
```
Comment (mgmax) [inline FabLabKasse/scriptHelper.py]: typo? or gender bullshit?

Comment (patkan) [inline FabLabKasse/scriptHelper.py]: Man muss natürlich richtig gendern und so.
Spaß beiseite, ich dachte mir man kann das als easter egg einbauen, du darfst es aber auch gerne ändern :-P

**Comment by patkan** [comment]
I have rebased this branch against current `development`.

**Comment by sedrubal** [comment]
Can we merge this? :)

**Comment by mgmax** [comment]
the 'dex Verantwortlichx' should still be changed, maybe just to 'den Verantwortlichen'.

**Comment by sedrubal** [comment]
I like this hidden ironical political and social criticism. But if you worry about that, I think its no problem to change this.

Sometime in the future we should also change such messages to english and use internationalization and translations...

**Comment by patkan** [comment]
we could try to implement a test, that checks whether the graphical except-hook is working. However, in sum this is probably not worth the time.

**Comment by sedrubal** [comment]
I think, we can merge this, too. Until 21:00

**Comment by patkan** [comment]
That's OK for me.

**Pull request merged**